packages/site-admin/package.json

```
@@ -13,7 +13,8 @@
 	"scripts": {
 		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
 		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json && copy-assets && copyfiles package.json dist/",
-		"prepack": "yarn run clean && yarn run build"
+		"prepack": "yarn run clean && yarn run build",
+		"storybook": "sb dev"
```

```suggestion
		"storybook": "sb dev -p 57862"
```

It's convenient to hardcode some kind of port number here so it doesn't open a new tab on each reboot.